### ISSUE TITLE
It translates everything

### ISSUE THREAD

**Issue by carman247** [justkawal/arb_translator#1]
Hi,

This is great, but there's a problem if you're using an .arb file that has blocks like this

```
  "establishmentVirtualKiosk": "{establishmentName} Virtual Kiosk",
  "@establishmentVirtualKiosk": {
    "description": "heading text",
    "type": "text",
    "placeholders": {
      "establishmentName": {}
    }
  },
```

The only part that needs translating in this is `Virtual Kiosk`. The placeholder `{establishmentName}` and everything else should be ignored from the translation.

It's not really an issue but more of a feature request I guess ... 

e.g.

English > Chinese would return a block like this

```
  "establishmentVirtualKiosk": "{establishmentName} 虚拟亭",
  "@establishmentVirtualKiosk": {
    "description": "heading text",
    "type": "text",
    "placeholders": {
      "establishmentName": {}
    }
  },
```

I guess it becomes difficult because that placeholder could appear anywhere in the string to be translated

e.g.

`this is some text {placeHolder} that needs translating`

**Comment by justkawal** [comment]
@carman247 You are right as placeholder might appear anywhere, 

You can try one approach as to make an intermediate arb file which is then translated and then after that u can convert it into your desired arb.

**Comment by justkawal** [comment]
@carman247

I would love to re-open this issue because Open Source Contributor **RexMa**: @Nolence has started working on pr for the desired functionality.

A big thanks to @Nolence. 🔥

**Comment by stargazing-dino** [comment]
This should be fixed by #2. @justkawal If you want, you can make me an uploader on pub.dev as well so I can publish the package after PRs which should take some extra work off your hands. I intend to work on some CI as well for this package later on

<img width="949" alt="Screen Shot 2021-03-15 at 11 28 08 AM" src="https://user-images.githubusercontent.com/29642168/111195225-a75b8380-8581-11eb-96ae-11b8664cc947.png">
